src/components/Form/FetchDocumentForm.jsx

```diff
@@ -54,30 +61,29 @@ const FetchDocumentForm = () => {
     }
   };
 
-  const formRowStyle = {
-    margin: '20px 0'
-  };
-
-  /* eslint-disable jsx-a11y/label-has-associated-control */
   return (
     <FormSection
       title="Search Document"
       state={state}
       statusType="Search status"
       defaultMessage="To be searched..."
     >
-      <form onSubmit={handleGetDocumentSubmission} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="search-doctype">Select document type to search: </label>
-          <DocumentSelection htmlId="search-doctype" />{' '}
-          <button disabled={state.processing} type="submit">
-            Get Document
-          </button>
-        </div>
-      </form>
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleGetDocumentSubmission}>
+          <InputLabel id="search-doctype">
```

Comment:
Already tested, this `<InputLabel>`'s `id` needs to be `htmlFor` to get rid of blue warnings in dev tools. Currently, it is not linked to what it's trying to label.